animalkingdom/src/animalkingdom/Birds.java

```diff
@@ -0,0 +1,38 @@
+package animalkingdom;
+
+// if you want to see whats happening here go to mammels, its the same thing
+
+public class Birds extends AnimalActions {
+	
+	public Birds(String name, int year, int food) {
+
+		this.name = name;
+		this.year = year;
+		this.food = food;
+	}
+
+	@Override
+	public String getName(){
+		return name;
+	};
+
+	@Override
+	public String getMoveType(){
+		return "feet and wings";
+	};
+
+	@Override
+	public String getBreatingType(){
+		return "lungs, airsack and larx";
+	};
```

Comment:
Be careful here - on line 25, there is a missing `'h'` character for your method name that will cause a compile time error.